src/compounds/RequestItem/RequestItem.jsx

```diff
@@ -35,11 +41,20 @@ const RequestItem = React.forwardRef(({ createdAt, description, href, img, title
 RequestItem.propTypes = {
   createdAt: PropTypes.string.isRequired,
   description: PropTypes.string.isRequired,
+  href: PropTypes.string,
   id: PropTypes.number,
+  index: PropTypes.number.isRequired,
   img: PropTypes.shape(Image.propTypes).isRequired,
   status: PropTypes.shape(Badge.propTypes).isRequired,
   title: PropTypes.string.isRequired,
   updatedAt: PropTypes.string.isRequired,
 }
 
+RequestItem.defaultProps = {
+  href: '',
+  id: null,
+}
+
+RequestItem.displayName = 'Request Item'
```

Comment:
what's this about?

Reply:
this was a linting error:
<img width="817" alt="image" src="https://user-images.githubusercontent.com/73361970/215606539-2c338729-0bd7-48b4-b0eb-2e69bbdb055f.png">

Reply:
it happens when the component has a ForwardRef
https://stackoverflow.com/questions/67992894/component-definition-is-missing-display-name-for-forwardref